_episodes/05-function-style.md

```diff
@@ -113,54 +144,56 @@ The docstring should explain what the function or module does (and not how it is
 >
 > When you add a docstring to a function or module, python automatically adds this to the `__doc__` attribute of the object.
 >
-> You can also see an object's docstring by typing `object.__doc__` into the Python interpreter. For example, to see the docstring associated with the canvas function, `molssi_devops.canvas.__doc__` into the Python interpreter (after importing `molssi_devops`, of course.)
+> You can also see an object's docstring by typing `object.__doc__` into the Python interpreter. For example, to see the docstring associated with the canvas function, `molecool.canvas.__doc__` into the Python interpreter (after importing `molecool`, of course.)
 {: .callout}
 
 ### Sections of a Docstring
-There are many ways you could format this docstring (different styles/conventions). We recommend using [numpy style docstrings], and this is what the example above and `mean` funtion are written in.
+There are many ways you could format this docstring (different styles/conventions). We recommend using [numpy style docstrings], and this is what the example above and `open_pdb` funtion are written in.
 
 Each docstring has a number of sections which are separated by headings. Headings should be underlined with hyphens (`-----`). There are many options for sections, we will only cover the most relevant here. If you would like to see a full list, check out the documentation for [numpy style docstrings].
 
 #### 1. Short summary
-A one-line summary that does not use the variable name or the function name. In our `mean` function, this corresponds to the following.
+A one-line summary that does not use the variable name or the function name. In our `open_pdb` function, this corresponds to the following.
 
 ~~~
 """
-Calculate the mean/average of a list of numbers.
+Open and read coordinates and atom symbols from a pdb file.
 """
 ~~~
 {: .language-python}
 
 #### 2. Extended summary
-A few sentences giving a detailed description of the function or module. This section should be used to clarify *functionality*, not to discuss implementation.  
+A few sentences giving a detailed description of the function or module. This section should be used to clarify *functionality*, not to discuss implementation.
 
-We do not have an extended summary in our `mean` function, since it is relatively straightforward.
+We do not have an extended summary in our `open_pdb` function, since it is relatively straightforward.
```

Comment:
We do have an extended summary in this case. It is

"The pdb file must specify the atom elements in the last column, and follow
     the conventions outlined in the PDB format specification"